packages/manager/modules/bm-server-components/src/network-tile/template.html

```diff
@@ -190,19 +190,19 @@
             >
                 <oui-action-menu-item
                     data-ng-if="!$ctrl.ola.isActivated()"
-                    data-ui-sref="app.dedicated-server.server.interfaces"
+                    data-ui-sref="{{ $ctrl.statePrefix }}.interfaces"
                 >
                     <span data-translate="server_network_ola_activate"></span>
                 </oui-action-menu-item>
                 <oui-action-menu-item
                     data-ng-if="$ctrl.ola.isConfigured()"
-                    data-ui-sref="app.dedicated-server.server.interfaces.ola-reset"
+                    data-ui-sref="{{ $ctrl.statePrefix }}.interfaces.ola-reset"
                 >
                     <span data-translate="server_network_ola_reset"></span>
                 </oui-action-menu-item>
                 <oui-action-menu-item
                     data-ng-if="$ctrl.ola.isActivated() && !$ctrl.ola.isConfigured()"
-                    data-ui-sref="app.dedicated-server.server.interfaces.ola-configuration"
+                    data-ui-sref="{{ $ctrl.statePrefix }}.interfaces.ola-configuration"
```

Comment:
```suggestion
                    data-ui-sref=".interfaces.ola-configuration"
```

Reply:
It's applied for all places where i use child state.